src/main.cpp

```diff
@@ -17,11 +17,11 @@ Napi::Object DDWAF::Init(Napi::Env env, Napi::Object exports) {
   mlog("Setting up class DDWAF");
   Napi::Function func = DefineClass(env, "DDWAF", {
     StaticMethod<&DDWAF::version>("version"),
-    InstanceMethod<&DDWAF::updateRuleData>("updateRuleData"),
-    InstanceMethod<&DDWAF::toggleRules>("toggleRules"),
+    InstanceMethod<&DDWAF::update>("update"),
     InstanceMethod<&DDWAF::createContext>("createContext"),
     InstanceMethod<&DDWAF::dispose>("dispose"),
     InstanceAccessor("disposed", &DDWAF::GetDisposed, nullptr, napi_enumerable),
+    InstanceAccessor("rulesInfo", &DDWAF::GetRulesInfo, nullptr, napi_enumerable),
```

Comment:
why do you need a getter ? why not just do a "setter" function, that sets the rulesInfo property everytime it changes. I don't see any processing done `DDWAF::GetRulesInfo`

Reply:
`InstanceAccessor` has been removed and applied the previous approach but without `PropertyDescriptor` (just using Set)